### PR TITLE
Add interfaces API endpoint documentation

### PR DESCRIPTION
**Pull request by yubiuser**
Documentation for https://github.com/pi-hole/FTL/pull/1355

### CODE REVIEW

**Review by DL6ER — APPROVED** [review]

**Review by rdwebdesign — COMMENTED** [review]
I have a **suggestion**:

Currently, this page uses List Items to show each API endpoint, like this:

```
- `>endpoint1`: explanation

    ```text
    output of endpoint1
    ```
```

I think we could replace the list items with Headings, like this:

```
    ## `>endpoint1`:
    explanation
    ```text
    output of endpoint1
    ```

    ## `>endpoint2`:
    Explanation 2
    ```text
    output of endpoint2
    ```
```

Using Headings the endpoints would be listed in the "Table of contents" sub-menu.

**Comment by yubiuser** [comment]
Force pushed to adhere the new heading style from https://github.com/pi-hole/docs/pull/729

**Comment by yubiuser** [comment]
Needs re-approval.

File: docs/ftldns/telnet-api.md [hunk 1]
```diff
@@ -247,4 +247,29 @@ Get the IP of the gateway of the default route and the corresponding interface
 
 Note that if no non-default route could be found, `0.0.0.0` and an empty interface string is returned
 
+#### `>interfaces` {data-toc-label='interfaces'}
```
Comment (rdwebdesign) [inline docs/ftldns/telnet-api.md:250]: You need to add the 
```

---

```

before the new `>interfaces` item.

**Review by rdwebdesign — CHANGES_REQUESTED** [review]

**Review by rdwebdesign — APPROVED** [review]

**Pull request merged**